Rakefile

```diff
@@ -114,12 +114,19 @@ MRuby.each_target do |target|
 end
 
 mkdir_p "#{MRUBY_ROOT}/build"
-cfiles = (Dir.glob("#{MRUBY_ROOT}/src/*.c")+
-          Dir.glob("#{MRUBY_ROOT}/mrbgems/**/*.c")+
-          Dir.glob("#{MRUBY_ROOT}/build/repos/**/{src,test,core}/*.c")).uniq
-rbfiles = (Dir.glob("#{MRUBY_ROOT}/{mrblib,test,test/t}/*.rb")+
-           Dir.glob("#{MRUBY_ROOT}/mrbgems/*/{mrblib,test}/*.rb")+
-           Dir.glob("#{MRUBY_ROOT}/build/repos/**/{mrblib,test}/*.rb")).uniq
+cfiles = (
+  Dir.glob("#{MRUBY_ROOT}/src/*.c")+
+  Dir.glob("#{MRUBY_ROOT}/mrbgems/**/*.c")+
+  Dir.glob("#{MRUBY_ROOT}/build/repos/**/{src,test,core}/*.c")
+).uniq
+rbfiles = (
+  Dir.glob("#{MRUBY_ROOT}/{mrblib,test,test/t}/*.rb")+
+  Dir.glob("#{MRUBY_ROOT}/{mrblib,test,test/t}/**/*.rb")+
```

Comment:
`**/*.rb` contains `*.rb` so you only need the former.

Reply:
Fixed!